nearswap/src/deposit.rs

```diff
@@ -52,19 +52,18 @@ impl NearSwap {
         let mut d = self.get_deposit(&sender);
         let amount = env::attached_deposit();
         d.ynear += amount;
-        self.deposits.insert(&sender, &d);
+        self.deposits.insert(&sender, &d.into());
```

Comment:
why do we use .into here ? is it related  to linter warnings ?

Reply:
because we wrap deposit into a variant. Look at this code: https://github.com/alpha-fi/clp-contracts/pull/102/files#diff-636244dcf34d0a121d2dadc127de50a63a3635feb4916e2eac1c445a78136eb7R165-R181

Reply:
got it